docs/README.rst

```diff
@@ -53,11 +53,11 @@ or just to run documentation building
 
      breeze build-docs --docs-only
 
-Also, you can only build one documentation via ``--package-filter``.
+Also, you can only build documentation for a subset of packages via ``--package-filter``. For example, to build documentation only for the main Airflow package and the Kubernetes provider package, you can run the following command:
 
 .. code-block:: bash
 
-    breeze build-docs --package-filter <PACKAGE-NAME>
+    breeze build-docs --package-filter apache-airflow --package-filter apache-airflow-providers-cncf-kubernetes
```

Comment:
This example was much needed :)

Reply:
You may want to update here as well - https://github.com/apache/airflow/blob/e6f445129a998eab62d71bd91b4a5f46cd77c1de/dev/breeze/src/airflow_breeze/commands/developer_commands.py#L370